INFO     lingkuan_828.commons.variable_manager:variable_manager.py:40 [2025-08-29 09:14] 成功加载静态变量: VAR\uat_data.json
INFO     lingkuan_828.commons.variable_manager:variable_manager.py:64 [2025-08-29 09:14] 成功加载运行时变量: VAR\runtime_vars_vps.json
INFO     lingkuan_828.conftest:conftest.py:62 [2025-08-29 09:22:20] 用例 test_vps/test_vps_ordersend2.py::TestVPSOrderSend_newScenarios::TestVPSOrderSend1::test_trader_orderSend 开始登录，环境: uat
INFO     lingkuan_828.conftest:conftest.py:100 UAT登录尝试 1/5，MFA验证码: 190092
INFO     lingkuan_828.conftest:conftest.py:124 UAT登录成功（第1次尝试）
INFO     lingkuan_828.commons.variable_manager:variable_manager.py:111 [2025-08-29 09:14] 运行时变量已保存到: D:\pycharm_test\lingkuan_828\VAR\runtime_vars_vps.json
INFO     lingkuan_828.conftest:conftest.py:152 切换到VPS URL: https://39.101.181.190/vps
INFO     lingkuan_828.commons.api_base:api_base.py:152 [2025-08-29 09:22:21] POST请求（JSON）: /subcontrol/trader/orderSend | 数据: {'symbol': 'XAUUSD', 'placedType': 0, 'remark': 'ces', 'intervalTime': 100, 'type': 0, 'totalNum': '', 'totalSzie': '1.00', 'startSize': '0.60', 'endSize': '1.00', 'traderId': 7214}
INFO     lingkuan_828.commons.api_base:api_base.py:160 [2025-08-29 09:22:21] 请求后等待 1 秒
ERROR    lingkuan_828.commons.api_base:api_base.py:423 [2025-08-29 09:22:22] JSON断言失败: Failed: 响应msg字段应为success（JSON路径值不匹配） | 路径: $.msg | 响应: {"code":500,"msg":"300151下单失败,该账号正在下单中","data":null}
INFO     lingkuan_828.conftest:conftest.py:157 [2025-08-29 09:22:23] 用例 test_vps/test_vps_ordersend2.py::TestVPSOrderSend_newScenarios::TestVPSOrderSend1::test_trader_orderSend 执行完毕
INFO     lingkuan_828.conftest:conftest.py:62 [2025-08-29 09:22:28] 用例 test_vps/test_vps_ordersend2.py::TestVPSOrderSend_newScenarios::TestVPSOrderSend1::test_trader_orderSend 开始登录，环境: uat
INFO     lingkuan_828.conftest:conftest.py:100 UAT登录尝试 1/5，MFA验证码: 190092
INFO     lingkuan_828.conftest:conftest.py:124 UAT登录成功（第1次尝试）
INFO     lingkuan_828.commons.variable_manager:variable_manager.py:111 [2025-08-29 09:14] 运行时变量已保存到: D:\pycharm_test\lingkuan_828\VAR\runtime_vars_vps.json
INFO     lingkuan_828.conftest:conftest.py:152 切换到VPS URL: https://39.101.181.190/vps
INFO     lingkuan_828.commons.api_base:api_base.py:152 [2025-08-29 09:22:29] POST请求（JSON）: /subcontrol/trader/orderSend | 数据: {'symbol': 'XAUUSD', 'placedType': 0, 'remark': 'ces', 'intervalTime': 100, 'type': 0, 'totalNum': '', 'totalSzie': '1.00', 'startSize': '0.60', 'endSize': '1.00', 'traderId': 7214}
INFO     lingkuan_828.commons.api_base:api_base.py:160 [2025-08-29 09:22:29] 请求后等待 1 秒
ERROR    lingkuan_828.commons.api_base:api_base.py:423 [2025-08-29 09:22:30] JSON断言失败: Failed: 响应msg字段应为success（JSON路径值不匹配） | 路径: $.msg | 响应: {"code":500,"msg":"300151下单失败,该账号正在下单中","data":null}
INFO     lingkuan_828.conftest:conftest.py:157 [2025-08-29 09:22:30] 用例 test_vps/test_vps_ordersend2.py::TestVPSOrderSend_newScenarios::TestVPSOrderSend1::test_trader_orderSend 执行完毕